{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "281f10b6_9de0dc81",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1000135
      },
      "writtenOn": "2022-04-30T18:37:47Z",
      "side": 1,
      "message": "Looks good to me. There seems to be an outage of our ci today so the submissions are not getting built, I expect that will get fixed in a while and when it does get build and passes ci we can push",
      "revId": "142e82989c84e6417d35563e900e87c1ad5727ec",
      "serverId": "6622c302-437a-49ad-a91d-55ba7dcf1615"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "d913585d_4791b1e1",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1003569
      },
      "writtenOn": "2022-04-30T22:52:30Z",
      "side": 1,
      "message": "Hello，I found a bug（https://bug-attachments.documentfoundation.org/attachment.cgi?id\u003d179423）, but don\u0027t know how to solve it. Can you help me solve it, or guide me to solve it?",
      "parentUuid": "281f10b6_9de0dc81",
      "revId": "142e82989c84e6417d35563e900e87c1ad5727ec",
      "serverId": "6622c302-437a-49ad-a91d-55ba7dcf1615"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "84079f4f_2cca8819",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1003569
      },
      "writtenOn": "2022-04-30T23:09:39Z",
      "side": 1,
      "message": "Sorry, the bug’s link is https://bugs.documentfoundation.org/show_bug.cgi?id\u003d148470 \nThe link in the previous reply is the attachment address and “)” are recognized as part of link.",
      "parentUuid": "d913585d_4791b1e1",
      "revId": "142e82989c84e6417d35563e900e87c1ad5727ec",
      "serverId": "6622c302-437a-49ad-a91d-55ba7dcf1615"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "c41a91fa_c6e587ec",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1000135
      },
      "writtenOn": "2022-05-02T16:54:15Z",
      "side": 1,
      "message": "Right now I don\u0027t know, I will try and have a look. I suspect the Fedora Linux problem is a different but similar problem",
      "parentUuid": "84079f4f_2cca8819",
      "revId": "142e82989c84e6417d35563e900e87c1ad5727ec",
      "serverId": "6622c302-437a-49ad-a91d-55ba7dcf1615"
    }
  ]
}